example/example.test.js

```diff
@@ -6,23 +6,39 @@ const filePath = './example/customJSFunctions.js';
 
 // Import and initialize
```

Comment:
I would remove this comment - it was part of a collection of a few comments, but all the others are gone, Instead, the comment above ("Do not change...") shoudl go directly before these two lines